spec/overrides/action_controller_instrumentation_spec.rb

```diff
@@ -72,6 +76,39 @@ class FakeAuthenticatedController < FakeController
     expect(payload[:honeycomb_metadata]).to include(flash_notice: 'Fired ze missiles.')
   end
 
+  describe 'when capturing exceptions by default' do
+    it 'should captures the exception metadata' do
+      caught = false
+      begin
+        subject.honeycomb_attach_exception_metadata do
+            raise RuntimeError, 'kaboom!'
+        end
+      rescue Exception
```

Comment:
Any reason to do `Exception` over `StandardError` in these tests? Not a huge deal, but I hate when tests swallow a well-timed <kbd>ctrl</kbd><kbd>c</kbd>.